reduced font size of Facebook btn

jessevdp opened the pull request:
Text inside Facebook button was still displaying weirdly on some pages.

Reducing its font size (again) fixes it.

Fixes issue #5 (again)

jessevdp committed:
reduced font size of Facebook btn

Text inside Facebook button was still displaying weirdly on some pages.

Reducing its font size (again) fixes it.

See issue #5

lesander commented:
This is a rather dirty hack. I'd rather have a different `em` size for each viewport.

jessevdp commented:
That's not how it used to be but probably a best practise. I'll look into it.

jessevdp committed:
created media queries

Implemented the breakpoints that are in my opinion necessary for the
right display. Aiming towards the biggest possible font size at every
breakpoint.

These lines of css are `important` to override everything else.

jessevdp committed:
deleted unnecessary line of css

Font size is determined in the scale file now. Which is where it
belongs in my opinion. It’s responsive.

jessevdp commented:
Think it's all ready to merge now. Take a look at the `font-size` yourself to see if you want to make any changes.

lesander commented on _sass/scale.scss:
```diff
@@ -110,3 +110,22 @@ nav {
+
+  /* responsive displays */
+  @media #{$mq-max-xsmall} {
+    font-size: 1rem !important;
```
Are all those `!important`s required? If I'm correct, just the media queries will override the `font-size`.

jessevdp replied:
I suppose they're not needed anymore, while working on it I hadn't deleted the old styles yet... But I guess a simple test should tell us the answer...

jessevdp replied:
Ah, the media query **is being overwritten** as you can see in the following screenshot. I'll delete the css that overwrites it as well as the `!important` tags.

(this only happens on the news page)

<img width="333" alt="screen shot 2017-01-21 at 14 51 14" src="https://cloud.githubusercontent.com/assets/15139826/22174870/54b37882-dfe9-11e6-9acf-fba9b106fce6.png">

jessevdp replied:
Wait a minute, can't really delete the css since it does not ***only*** affect the fb button but all links on the page.

lesander replied:
I suppose you could create a new css selector specifically for the affected buttons.

jessevdp commented:
Ah, the media query **is being overwritten** as you can see in the following screenshot. I'll delete the css that overwrites it as well as the `!important` tags.

(this only happens on the news page)

<img width="333" alt="screen shot 2017-01-21 at 14 51 14" src="https://cloud.githubusercontent.com/assets/15139826/22174870/54b37882-dfe9-11e6-9acf-fba9b106fce6.png">

jessevdp committed:
removed `!important` where unnecessary

By adding a `:not(.btn-fb)` selector to the css that was overwriting
the media queries the `!important`’s are no longer needed.

It does work! :smile:
